rootfs/scheduler/mock.py

```diff
@@ -57,6 +57,18 @@ def stop(self, name):
         job['state'] = JobState.down
 
     # Related to k8s services
+    def resolve_state(self, pod):
+        # See "Pod Phase" at http://kubernetes.io/v1.1/docs/user-guide/pod-states.html
+        states = {
+            "Pending": JobState.initialized,
```

Comment:
just tried it, terminating pods just don't show up in the API anymore. Did a quick look only tho